### DISCUSSION TITLE
Maximum Plans

### DISCUSSION THREAD

**Discussion by joesixpack** [subquery/subql#954] [Frontier TestNet - Bug Reporting]
I was able to create more than five "maximum" plans, i.e. six.

**Comment by ianhe8x** [comment]
What's your indexer address? 
Five plans is for each deployment (empty deployment is considered as  default plan).
So totally you can have more than five.

**Reply by joesixpack** [reply]
This is true, it was six total spread over two deployments (now deleted).  I guess it wasn't too clear from the title since each deployment section didn't have that title, so assumed it was for all.

**Comment by mzxyz** [comment]
@lilian79511 @CaiYiLiang

**Reply by lilian79511** [reply]
Noted.
